GliaWidgets/Sources/File/LocalFile.swift

```diff
@@ -53,9 +53,14 @@ extension LocalFile {
             completion(thumbnail)
             return
         } else {
-            environment.thumbnailGenerator.generateBestRepresentation(size, url.standardizedFileURL) { [weak self] image in
+            environment.thumbnailGenerator.createImageThumbnail(
+                fileURL: url,
+                size: size,
+                scale: environment.uiScreen.scale()
+            ) { [weak self] image in
+                let cropped = image?.resized(to: size)
                 self?.environment.gcd.mainQueue.async {
```

Comment:
Is it still necessary to use thread hop like `mainQueue.async`?

Reply:
Yes, because `QLThumbnailGenerator` return representation on background thread